docs/source/development/config.rst

```diff
@@ -41,8 +41,8 @@ The next thing you need to know is what to call your configuration file. The
 basic idea is that each application has its own default configuration filename.
 The default named used by the :command:`ipython` command line program is
 :file:`ipython_config.py`, and *all* IPython applications will use this file.
-Other applications, such as the parallel :command:`ipcluster` scripts or the
-QtConsole will load their own config files *after* :file:`ipython_config.py`. To
+Other applications, 
```

Comment:
This generality is a bit confusing now, because there are only two applications we maintain that use this, and the kernel isn't really what most people would think of as an application.

Maybe be more specific:

> The IPython kernel will load its own config file *after* `ipython_config.py`.